panc/src/main/scripts/panlint/panlint.py

```diff
@@ -33,6 +33,18 @@ def __init__(self, filename, number, text):
         self.filename = str(filename)
         self.number = int(number)
         self.text = str(text)
+        self.problems = []
+
+
+class Problem(object):
+    def __init__(self, start, end, message):
```

Comment:
What do start and end represent? They are ints so I guess positions in the line? It would be nice to make that explicit. Same for message: I can probably guess what it means by looking at usage but handy to have a doc.